public/js/angular/controllers.js

```diff
@@ -377,7 +377,9 @@ angular
       $scope.reverse = false;
       $scope.predicate = 'created';
       $scope.hasPermissions = function (badge) {
-        return window.badgesPermissionsModel({
+        var badgesPermissionsModel = module.exports;
```

Comment:
it feels like this should be possible as an include based on what's loaded up top. @gvn do you have any wisdom here?